src/pages/stacks-blockchain/overview.md

```diff
@@ -50,28 +50,28 @@ Smart contracts on the Stacks 2.0 blockchain are written in a new language calle
 
 ### Stacks Mining
 
-Mining is required to make the network trustworthy and secure, by verifying transaction information. Mining is the act of participation in the consensus mechanism of the Stacks 2.0 blockchain, called Proof of Transfer (PoX). To incentivize miners, who have to run mining software on their machine, they receive freshly minted Stacks (STX) tokens.
+Mining is required to make the network usable, trustworthy, and secure. Miners verify incoming transactions and participate in the consensus mechanism of the Stacks 2.0 blockchain, called Proof of Transfer (PoX). To incentivize mining, miners receive freshly minted Stacks (STX) tokens.
 
 [@page-reference | inline]
 | /mining
 
 ### Proof of Transfer (PoX)
 
-Proof of Transfer (PoX) is the consensus mechanism that leverages the security and stability of Bitcoin to create new blockchains, without modifying the Bitcoin protocol.
+Proof of Transfer (PoX) is a novel consensus mechanism that leverages the security and stability of Bitcoin to create a new blockchain, without modifying the Bitcoin protocol. 
 
-PoX makes it possible to reward holders of Stacks Token (STX) with Bitcoin: instead of spending energy, Stacks miners transfer Bitcoin to holders of Stacks Token (STX) to mine blocks. An implementation of this is called **Stacking**.
+STX miners transfer Bitcoin to other STX holders in order to mine blocks. This BTC reward mechanism is called **Stacking**.
```

Comment:
We agreed a few weeks ago to always use `Stacks (STX)` when we reference the token. I'm seeing this inconsistency in different places in your PR. Could you adapt to the convention, please? :)